docs/root/install/getting_started.rst

```diff
@@ -0,0 +1,61 @@
+.. _install_getting_started:
+
+Getting Started
+===============
+
+This section features sample configurations and guides to quickly getting started with Envoy.
+
+A basic configuration
+---------------------
+
+A very minimal Envoy configuration that can be used to validate basic plain HTTP
+proxying is available in :repo:`configs/google_com_proxy.json`. This is not
```

Comment:
Can we reference the new v2 YAML config?

Reply:
Yes, I can do that. There are two changes that I need to make to the default YAML to get it work for me -- I need to set `dns_lookup_family` to do V4_ONLY and change the listener to 0.0.0.0. If these changes are OK to put into the default smoketest, I'm happy to create a PR for this.

Reply:
SGTM. the V4_ONLY is already there but commented out. Let's just make it the default for smoke test purposes (maybe with a comment there that talks about how to test v6 if desired).